client/packages/invoices/src/StockOut/utils.tsx

```diff
@@ -93,28 +93,43 @@ export const createDraftStockOutLineFromStockLine = ({
 export const createDraftStockOutLine = ({
   invoiceLine,
   stockLine,
```

Comment:
Wouldn't invoiceLine always have stockLine ?

Reply:
Yeah I'm a little confused by the reasoning for this being passed as well, as the `invoice.stockLine` which should already exist...
Maybe better to remove it to remove any confusion...

Reply:
Was scared to remove it and find somewhere it was needed but missed in testing.
That being said it feels like it's unnecessary, maybe some state thing when only one of the queries is refreshed but even then it's the invoice one that should matter right?

Reply:
Yeah could be true, I am ok to keep, especially since we will be refactoring this area